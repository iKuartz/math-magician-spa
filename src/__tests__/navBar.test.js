import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from '../components/Nav';

it('Checking if Navigation Bar renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <NavBar />
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
