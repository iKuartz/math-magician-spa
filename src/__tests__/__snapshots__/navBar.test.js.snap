// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking if  1`] = `
<nav
  className="navBar"
>
  <h1
    className="header"
  >
    Math Magicians
  </h1>
  <ul
    className="ul-links"
  >
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        aria-current="page"
        className="links active"
        exact={true}
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        className="links"
        exact={true}
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        className="links"
        exact={true}
        href="/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </li>
  </ul>
</nav>
`;

exports[`Checking if Navigation Bar re 1`] = `
<nav
  className="navBar"
>
  <h1
    className="header"
  >
    Math Magicians
  </h1>
  <ul
    className="ul-links"
  >
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        aria-current="page"
        className="links active"
        exact={true}
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        className="links"
        exact={true}
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        className="links"
        exact={true}
        href="/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </li>
  </ul>
</nav>
`;

exports[`Checking if Navigation Bar renders correctly 1`] = `
<nav
  className="navBar"
>
  <h1
    className="header"
  >
    Math Magicians
  </h1>
  <ul
    className="ul-links"
  >
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        aria-current="page"
        className="links active"
        exact={true}
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        className="links"
        exact={true}
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        className="links"
        exact={true}
        href="/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </li>
  </ul>
</nav>
`;

exports[`renders correctly 1`] = `
<nav
  className="navBar"
>
  <h1
    className="header"
  >
    Math Magicians
  </h1>
  <ul
    className="ul-links"
  >
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        aria-current="page"
        className="links active"
        exact={true}
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        className="links"
        exact={true}
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        className="links"
        exact={true}
        href="/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </li>
  </ul>
</nav>
`;
